zips/zip-0228.rst

```diff
@@ -2,12 +2,539 @@
+|``32``                       |``bsk``                   |``byte[32]``                               |The binding signature key, corresponding to the sum of all the       |  
+|                             |                          |                                           |:math:`\mathsf{rcv}` values of commitments in the Order.             |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``varies``                   |``SwapActionGroup``       |``ActionGroupDescription``                 |The description of the Action Group corresponding to the desired     |  
+|                             |                          |                                           |Swap operation. It must contain at least two Actions, the Swap       |  
+|                             |                          |                                           |Action and the Fee Action.                                           |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+The ``SwapIO`` data type is used to represent the input and output of a Swap Order.
+
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                                 | Description                                                         |
++=============================+==========================+===========================================+=====================================================================+
+|``32``                       |``asset_base``            |``byte[32]``                               |The Asset Base [#zip-0227-assetbase]_ of the input/output of the     |  
+|                             |                          |                                           |Swap Order.                                                          |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``8``                        |``qty``                   |``uint64``                                 |The quantity of the Asset represented by ``asset_base`` proposed in  |  
+|                             |                          |                                           |the Swap Order.                                                      |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+Rationale for Swap Orders
```

Comment:
The main rational is the fact that actions are shielded and to come up with a matching actionGroup, party A must revel the content of the offer. (this is relevent even without matcher in p2p context)

Reply:
Added some material to the start of the section